src/fu-engine.c

```diff
@@ -1009,8 +1020,10 @@ fu_engine_require_vercmp (XbNode *req, const gchar *version, GError **error)
 	} else if (g_strcmp0 (tmp, "ge") == 0) {
 		rc = fu_common_vercmp (version, version_req);
 		ret = rc >= 0;
+#ifdef HAVE_FNMATCH_H
 	} else if (g_strcmp0 (tmp, "glob") == 0) {
 		ret = fnmatch (version_req, version, 0) == 0;
+#endif
```

Comment:
Is there an else condition that could avoid breaking glob?

Reply:
I might just write a really dumb version for the common case of `something*` and `*something`